cmd/ursrv/formatting.go

```diff
@@ -112,16 +112,17 @@ func commatize(sep, s string) string {
 	return b.String()
 }
 
-func proportion(m map[string]int, count int) float64 {
+func proportion(m map[string]intString, count int) float64 {
```

Comment:
I don't we can do privacy stuff like this in here (or other places that deal with ratios). Consider for example a case where we have 11 connections that are IPv4 and 1 connection IPv6. Currently we'd (correctly) report that as 92% IPv4 and 8% IPv6. With privacy adjustments we get 11 IPv6 connections and 10 IPv4 connections, or 52%/48%.

Reply:
1. Currnently any group where this applies the change from e.g. 3 to 10 has no statistical impact. All groups where the ratios are calculated have sizes more than several thousands. Some of the groups in those ratios, however, are <10.
2. If the ratio is indeed changed by this small increase in counts, the group size is so small that it shouldn't be published anyway. It is either too small for statistical significance (i.e. unremarkable) or too small to be private anyway. This point is made null by point 1.

If you don't want to mess with ratio calculation, I think the small numbers should be omitted entirely instead. The numbers would be small enough that it doesn't really matter.